app/Http/Controllers/PollerGroupController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Device;
+use App\Models\PollerGroups;
+use Illuminate\Http\Request;
+
+class PollerGroupController extends Controller
+{
+    public function destroy(Request $request, PollerGroups $pollergroup)
+    {
+        $this->authorize('admin', $request->user());
```

Comment:
Try to make specific authorization, which can just defer to isAdmin() for now in the authorization code.  This will save us time down the road when we implement RBAC.

Reply:
changed